testbed/exercism-python/exercises/practice/accumulate/accumulate_test.py

```diff
@@ -9,31 +9,33 @@ def test_empty_sequence(self):
 
     def test_pow(self):
```

Comment:
We can't let the actual accumulate get formatted because it could make the saved gpt output invalid but I'm inclined to let our linter change the test.